tests/tests/documentation.rs

```diff
@@ -1892,6 +1903,7 @@ fn schema(mut conn: PgConnection) {
 }
 
 #[rstest]
+#[ignore = "paradedb.index_size deprecated"]
```

Comment:
Should we just get rid of this? I don't think it's even documented anywhere in our `/docs`. I had created a GitHub Issue to document it, but perhaps we can just get rid of it altogether now.